### PR TITLE
Make particle literals available from top-level import

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Fixes https://github.com/scikit-hep/particle/issues/238.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
Now you can do
```
In [1]: from particle import literals as lp

In [2]: from particle.literals import Lambda_b_0
```

File: src/particle/__init__.py [hunk 1]
```diff
@@ -17,6 +19,11 @@
 from .particle import Particle, SpinType, Parity, Charge, Inv, Status
 from .particle import ParticleNotFound, InvalidParticle
 
+# Direct access to Particle literals
+from .particle import literals
+
+sys.modules["particle.literals"] = literals
```
Comment (HDembinski) [inline src/particle/__init__.py:25]: For my own education, why is this line needed?

**Comment by eduardo-rodrigues** [comment]
> Looks great! I suppose you also test somewhere that from particle.pdgid import literals still works as before.

Correct, see `tests/pdgid/test_literals.py`.

File: src/particle/__init__.py (hunk 1 above)
Comment (eduardo-rodrigues) [reply src/particle/__init__.py:25]: This is needed for `from particle.literals import ...` to work if I do not move the submodule around. Indeed the statement expects `particle.literals` to be a file/module, and it isn't strictly speaking since at `particle.particle.literals` in reality. Indeed I thought I would keep the structure and old functionality. This line is needed at least to the best of my knowledge to make both statements work. Without it `from particle import literals as lp` would work but not `from particle.literals import Lambda_b_0`.

Comment (HDembinski) [reply src/particle/__init__.py:25]: Ah, I see, so this is a pretty clever solution. Thanks for the nice explanation.

Comment (henryiii) [reply src/particle/__init__.py:25]: :+1:

**Comment by eduardo-rodrigues** [comment]
Thanks both!
